fuel-vm/src/interpreter/balances.rs

```diff
@@ -103,22 +105,32 @@ impl RuntimeBalances {
     /// Note: This will not append a new asset into the set since all the assets must be created
     /// during VM initialization and any additional asset would imply reordering the memory
     /// representation of the balances since they must always be ordered, as in the protocol.
-    pub fn checked_balance_add(&mut self, memory: &mut [u8], asset: &AssetId, value: Word) -> Option<Word> {
+    pub fn checked_balance_add(
```

Comment:
I don't think there's a usecase for adding to the initial balances. We discussed removing this method, was that done in a separate PR?

Reply:
Yeh I think it's in the next pr

Reply:
Do we need to create an issue to track it?

Reply:
It's already fixed in the next PR